cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -124,13 +123,7 @@
 
     <!--[if IE 9]><script src="{{ static('js/ie/common.ie.js') }}"></script><![endif]-->
 
-    {# TODO: jQuery CDN is provided in the head to satisfy
-             Google Tag Manager (GTM) requirements.
-             Ideally GTM would handle its own dependency management
-             and initializing jQuery could be moved to the footer. #}
-
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script>if (!window.jQuery) { document.write('<script src="{{ static('js/jquery.min.js') }}"><\/script>'); }
+    <script src="{{ static('js/jquery.min.js') }}"></script>
```

Comment:
Have we confirmed with the data team that they no longer require CDN-loaded jQuery in the `head`? And if they do, how do we reconcile that with the directive from @ascott1?

If they don't need it, then we should proceed, but move the jQuery inclusion to the bottom of `body`, right?